### ISSUE TITLE
Create Scroll UX improvements

### ISSUE THREAD

**Issue by rvorias** [BibliothecaForAdventurers/realms-react#129]

**Comment by rvorias** [comment]
1. There's a create scroll button at the end of the md editor:
![image](https://user-images.githubusercontent.com/21060408/167587686-5d517a07-79f5-4688-bdbc-d5bc03f1a1ce.png)
2. Suggestion: clicking this would put the focus to the PREVIEW md renderer and another "CREATE SCROLL" button. Motivation: this let's the user read the preview and check if everything is ok.
3. The No results, Clear filters and See all realms components should not be at the bottom of the previewer 
![image](https://user-images.githubusercontent.com/21060408/167588042-d8b6e063-c067-410f-88b4-954ecf63cf45.png)

**Comment by ponderingdemocritus** [comment]
Yes great idea, all for it

**Comment by lordabrax** [comment]
Yes, I spotted it as well but was focusing on Revisions. If you fix it - that would be perfect!

**Comment by rvorias** [comment]
I did not implement an inbetween button directly under the editor. This because hovering off the button causes a state refresh and it pulls in all data from graphql
